src/main.tsx

```diff
@@ -13,7 +14,9 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
   <HashRouter>
     <Provider store={store}>
       <ThemeContextProvider>
-        <App />
+        <GoogleOAuthProvider clientId="493125079778-45igjn7jp3rtik2oc5pkv6rk2btp24tv.apps.googleusercontent.com">
```

Comment:
move clientId to `.env` files
use `dotenv` package
https://www.npmjs.com/package/dotenv